src/Aquifer.Jobs/Subscribers/UploadResourceContentAudioMessageSubscriber.cs

```diff
@@ -0,0 +1,421 @@
+﻿using System.Diagnostics;
+using System.Text;
+using Aquifer.Common.Configuration;
+using Aquifer.Common.Messages;
+using Aquifer.Common.Messages.Models;
+using Aquifer.Common.Services;
+using Aquifer.Common.Utilities;
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using Aquifer.Data.Enums;
+using Aquifer.Data.Schemas;
+using Aquifer.Data.Services;
+using Azure.Storage.Queues.Models;
+using CaseConverter;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Aquifer.Jobs.Subscribers;
+
+public sealed class CdnOptions
+{
+    public required string AquiferContentContainerName { get; init; }
+    public required string CdnBaseUri { get; init; }
+}
+
+public sealed class FfmpegOptions
```

Comment:
I could see moving all of the Ffmpeg stuff into its own service in the future.  I decided to not do that refactor yet because it's only used in this one place.  However, I organized the code to make it easier to pull things out in the future.